pkg/service/v0/graph.go

```diff
@@ -38,14 +40,12 @@ func (g Graph) UserCtx(next http.Handler) http.Handler {
 		if userID := chi.URLParam(r, "userID"); userID != "" {
 			user, err = g.ldapGetUser(userID)
 		} else {
-			// TODO: we should not give this error out to users
-			// http.Error(w, err.Error(), http.StatusInternalServerError)
-			render.Status(r, http.StatusNotFound)
+			errorcode.InvalidRequest.Render(w, r, http.StatusBadRequest)
```

Comment:
You should directly map the http status code instead of providing it here as well.

Reply:
According to the spec there is no direct relationship between the error code and the http status ...